src/SFA.DAS.Commitments.Infrastructure/Services/ApprenticeshipInfoService.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System.Linq;
+using System.Threading.Tasks;
+
+using SFA.DAS.Apprenticeships.Api.Client;
+using SFA.DAS.Commitments.Application.Interfaces;
+using SFA.DAS.Commitments.Domain.Entities.TrainingProgramme;
+using SFA.DAS.Commitments.Domain.Interfaces;
+
+namespace SFA.DAS.Commitments.Infrastructure.Services
+{
+    public class ApprenticeshipInfoService : IApprenticeshipInfoService
+    {
+        private const string StandardsKey = "Standards";
+        private const string FrameworksKey = "Frameworks";
+
+        private readonly ICache _cache;
+        private readonly IApprenticeshipInfoServiceConfiguration _config;
+        private readonly IApprenticeshipInfoServiceMapper _mapper;
+
+        public ApprenticeshipInfoService(ICache cache,
+            IApprenticeshipInfoServiceConfiguration config,
+            IApprenticeshipInfoServiceMapper mapper)
+        {
+            _cache = cache;
+            _config = config;
+            _mapper = mapper;
+        }
+
+        public async Task<StandardsView> GetStandardsAsync(bool refreshCache = false)
```

Comment:
Async in the method name is not needed.

Reply:
Removed 'Async' suffix from methods in this class.
(Also, optimised and tidied up GetTrainingProgram method.)